bot/exts/evergreen/trivia_quiz/_cog.py

```diff
@@ -0,0 +1,137 @@
+import json
+import logging
+import random
+import typing as t
+from pathlib import Path
+
+from discord.ext import commands, tasks
+
+from bot import constants
+from bot.bot import Bot
+from bot.exts.evergreen.trivia_quiz import _constants
+from bot.exts.evergreen.trivia_quiz import _helpers
+from bot.exts.evergreen.trivia_quiz._game import Game
+
+logger = logging.getLogger(__name__)
+
+p = Path("bot", "resources", "evergreen", "trivia_quiz.json")
+with p.open(encoding="utf8") as json_data:
+    QUESTIONS = json.load(json_data)
+
+
+class TriviaQuiz(commands.Cog):
+    """A cog for all quiz commands."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        self.wiki_questions: t.List = []
+        self.games: t.Dict = {}
+
+        self.get_wiki_questions.start()
+
+    def cog_unload(self) -> None:
+        """Cancel `get_wiki_questions` task when Cog will unload."""
+        self.get_wiki_questions.cancel()
+
+    @tasks.loop(hours=1.0)
+    async def get_wiki_questions(self) -> None:
+        """Get 10 random questions from wikipedia and format them like the questions in resources/."""
+        self.wiki_questions = []
+        start_id = 201
+
+        while len(self.wiki_questions) != 10:
+            async with self.bot.http_session.get(url=_constants.RANDOM_WIKI_URL) as r:
+                if r.status != 200:
+                    continue
+
+                raw_data = await r.json()
+
+                if any(not c.isalnum() for c in raw_data["title"]):
```

Comment:
```suggestion
                if all(c.isalnum() for c in raw_data["title"]):
```
I think this looks better, unless you want it to continue if `raw_data["title"]` is empty (if that's even possible).

Reply:
Oops it might need a `not` too.